### ISSUE TITLE
Is the server usually run in the background?

### ISSUE THREAD

**Issue by pathai-cshah** [kantord/SeaGOAT#238]
I run the server in tmux so that its running in the background and detach tmux but then gt complains that no server is running

**Comment by kantord** [comment]
Yeah, the server should keep running in the background! I don't know why it would stop running if you detach tmux. What exactly happens to whatever is running when you detach tmux?

**Comment by pathai-cshah** [comment]
It keeps running inside tmux, `gt` just errors out saying no server found

**Comment by kantord** [comment]
`gt` connected to the server through HTTP, but at the moment SeaGOAT is designed to be user-specific, basically `gt` will try to connect to the server that is running under the same user, there could be some further tricks required to share a SeaGOAT server with another user. (Probably a config file option could be added for connecting to the server of a different user, as well as manually specifying the URL if needed)

Could this be the explanation? I mean, are you using 2 different users? :thinking:

**Comment by Ekestfa** [comment]
I have the same problem. I installed the application via WSL. I cannot access the open session with `& disown`, there is no access to the open server.

**Comment by kantord** [comment]
> I have the same problem. I installed the application via WSL. I cannot access the open session with `& disown`, there is no access to the open server.

What would be the ideal way to solve that?
